src/test/scala/com/cognite/sdk/scala/common/ReadBehaviours.scala

```diff
@@ -175,4 +178,51 @@ trait ReadBehaviours extends Matchers { this: FlatSpec =>
       }
     }
   }
+
+  def readableWithRetrieveUnknownIds[R <: WithExternalId with WithId[Long], W](
+    readable: Readable[R, Id] with RetrieveByExternalIdsWithIgnoreUnknownIds[R, Id]
+                              with RetrieveByIdsWithIgnoreUnknownIds[R, Id]
+  ): Unit = {
+    // TODO: this test is not very stable as the fetched item may be deleted before it is fetched again by the external id
```

Comment:
To make this less likely to happen, perhaps we can add a filter on the creation time (should be older than 2020-03-01, say), so we don't end up reading recent items that may be about to be deleted in concurrent test runs?

Reply:
That's a good idea. We are not using a proper API filter (since it's a generic Readable), so I'll reduce it to "older than 10 minutes", so that we don't fetch thousands of rows before getting to something we want.